docs/source/contributing.rst

```diff
@@ -3,79 +3,79 @@
+repository. While this library is maintained by Ansys and
+submissions will be reviewed thoroughly before merging, we
+seek to foster a community that can support user questions
+and develop new features to make this libray a useful tool
+for all users. As such, we welcome and encourage any
+questions or submissions to this repository.
 
 
 Cloning the Source Repository
 -----------------------------
 
-You can clone the source repository from `DPF-Core
-GitHub <https://https://github.com/pyansys/DPF-Core>`_
+You can clone the ``pydpf-core`` repository from
+`GitHub <https://github.com/pyansys/pydpf-core>`_
 and install the latest version in development mode by running:
 
 .. code::
 
-    git clone https://github.com/pyansys/DPF-Core DPF-Core
-    cd DPF-Core
+    git clone https://github.com/pyansys/pydpf-core PyDPF-Core
+    cd PyDPF-Core
     pip install -e .
 
 
 Questions
 ---------
-For general or technical questions about the project, its
-applications, or about software usage, please create an issue at
-`DPF-Core Issues <https://github.com/pyansys/DPF-Core/issues>`_ where the
-community or DPF-Core developers can collectively address your
-questions.  The project support team can be reached at
-[alexander.kaszynski@ansys.com](mailto:alexander.kaszynski@ansys.com)
-
-By posting on the issues page, your question can be addressed by
-community members with the needed expertise and the knowledge gained
+For general or technical questions about this library, its
+applications, or usage, create an issue at
+`PyDPF-Core Issues <https://github.com/pyansys/pydpf-core/issues>`_,
+where the community or PyDPF-Core developers can collectively address
+your questions. The project support team can be reached at
+[alexander.kaszynski@ansys.com](mailto:alexander.kaszynski@ansys.com).
+
+By posting an issue, your question can be addressed by community
+members with the needed expertise and the knowledge gained
 will remain available on the issues page for other users.
 
 
 Reporting Bugs
 --------------
-If you encounter any bugs or crashes while using DPF-Core, please
-report it at `DPF-Core Issues <https://github.com/pyansys/DPF-Core/issues>`_
-with an appropriate label so we can promptly address it.  When
-reporting an issue, please be overly descriptive so that we may
-reproduce it. Whenever possible, please provide tracebacks,
+If you encounter any bugs or crashes while using PyDPF-Core, report
+it at `PyDPF-Core Issues <https://github.com/pyansys/pydpf-core/issues>`_
+with an appropriate label so we can promptly address it. When
+reporting an issue, be overly descriptive so that we can
+reproduce it. Whenever possible, provide tracebacks,
 screenshots, and sample files to help us address the issue.
 
 
 Feature Requests
 ----------------
-We encourage users to submit ideas for improvements to DPF-Core!
-Please create an issue on the `DPF-Core Issues <https://github.com/pyansys/DPF-Core/issues>`_
```

Comment:
Ansys Style Guide (and generally all tech style guides) say to avoid "please" in technical documentation.